src/java.base/share/classes/jdk/internal/foreign/ConfinedSession.java

```diff
@@ -41,8 +41,7 @@ final class ConfinedSession extends MemorySessionImpl {
 
     private int asyncReleaseCount = 0;
 
-    static final VarHandle ASYNC_RELEASE_COUNT= MhUtil.findVarHandle(
-            MethodHandles.lookup(), "asyncReleaseCount", int.class);
+    static final VarHandle ASYNC_RELEASE_COUNT= MhUtil.findVarHandle(MethodHandles.lookup(), "asyncReleaseCount", int.class);
```

Comment:
Since you are here, maybe you want to use `MHUtil::findVarHandle` also for the state var handle (for uniformity)

Reply:
I'm a little confused, I use `MHUtil::findVarHandle` for `acquireCount`, there is no need for a `VarHandle` for `state`.

Reply:
Apologies, I got confused.